tests/queries/0_stateless/01502_long_log_tinylog_deadlock_race.sh

```diff
@@ -88,3 +88,5 @@ test_with_engine Log
 
 $CLICKHOUSE_CLIENT -q "DROP TABLE IF EXISTS t1"
 $CLICKHOUSE_CLIENT -q "DROP TABLE IF EXISTS t2"
+
+clickhouse_test_wait_queries 60
```

Comment:
I guess if you will replace `timeout` with `clickhouse_client_loop_timeout`/`clickhouse_client_timeout`, `clickhouse_test_wait_queries` will not be required.

`clickhouse_test_wait_queries` should be used only when it is not possible to avoid leaving queries w/o it.